src/kOS/Module/kOSVesselModule.cs

```diff
@@ -228,20 +227,31 @@ private void ClearParts()
         /// </summary>
         private void AddDefaultParameters()
         {
-            AddFlightControlParameter("steering", new SteeringManager(parentVessel));
+            if (flightControlParameters != null)
+            {
+                flightControlParameters.Clear();
+            }
+            flightControlParameters = new Dictionary<string, IFlightControlParameter>();
+            AddFlightControlParameter("steering", new SteeringManager(Vessel));
+            AddFlightControlParameter("throttle", new ThrottleManager(Vessel));
+            AddFlightControlParameter("wheelsteer", new WheelSteeringManager(Vessel));
```

Comment:
Should say "wheelsteering", as that's what this flight control parameter is called elsewhere.  This was making ``lock wheelsteering`` fail with a misleading error message that there is no such control variable.